portal-web/test/functional/com/liferay/portalweb/macros/MessageboardsThread.macro

```diff
@@ -22,6 +22,8 @@
 	</command>
 
 	<command name="addCPWithAttachment">
+		<var name="key_attachmentFileName" value="${mbAttachmentFile}" />
```

Comment:
@acbbennett Where is this key variable being used in the macro?

Key variables should be declared right before the function that uses it.